index.html

```diff
@@ -129,49 +130,49 @@ <h4 class="let-s-connect-2">LET'S CONNECT!</h4>
                   </div>
                 </div>
                 <hr class="solid">
-              </li>
-            </ul>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+    <section class="contact-me">
+      <footer>
+        <div class="blue-rounded" id="contact">
+          <h3 class="contact-heading">Contact me</h3>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form id="contact-me" action="https://formspree.io/f/xgedddaq" method="post">
+            <!-- <label for="name"> Your name: </label> -->
+            <input id="name" type="text" name="name" placeholder="Enter your name" maxlength="30" required><br>
+            <!-- <label for="email"> Your email: </label> -->
+            <input id="mail" type="email" name="email" placeholder="Enter your email"><br>
+            
+            <!-- <label for="message"> Your message:</label> -->
+            <textarea id="message" name="message" cols="25" rows="20" placeholder="Write your message here" maxlength="500" required></textarea>
```

Comment:
- Optional
The placeholder for your text area field is not correct. Consider fixing it.
![image](https://user-images.githubusercontent.com/33268927/175763727-809e2f23-049f-4019-a103-e2e1104d6b24.png)